src/client/browser/idle-page/index.js

```diff
@@ -1,5 +1,28 @@
-import IdlePage from './idle-page';
+import * as browser from '../';
+import StatusIndicator from './status-indicator';
+import COMMAND from '../../../browser-connection/command';
 
-window.init = function (statusUrl) {
-    new IdlePage(statusUrl);
-};
+
+const CHECK_STATUS_DELAY = 1000;
+
+
+class IdlePage {
+    constructor (statusUrl, heartbeatUrl) {
+        this.statusIndicator = new StatusIndicator();
+
+        browser.startHeartbeat(heartbeatUrl, window.XMLHttpRequest);
+        this._checkStatus(statusUrl);
+    }
+
+    _checkStatus (statusUrl) {
```

Comment:
Argh, just realized that we can turn `statusUrl`  into the `IdlePage` field and don't pass it around.